vhdbuilder/packer/install-dependencies.sh

```diff
@@ -153,6 +153,9 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
     downloadCrictl ${CRICTL_VERSION}
     echo "  - crictl version ${CRICTL_VERSION}" >> ${VHD_LOGS_FILEPATH}
   done
+  
+  KUBERNETES_VERSION=$(echo -e "$CRICTL_VERSIONS" | head -n 1) installCrictl || exit $ERR_CRICTL_DOWNLOAD_TIMEOUT
```

Comment:
Do we need to handle separate installation of `crictl` for docker vs containerd builds here?

Reply:
shouldn't really matter since docker will use...docker I suppose